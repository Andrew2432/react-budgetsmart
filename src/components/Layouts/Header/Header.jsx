import React from 'react';
import { Navbar } from 'react-materialize';

const Header = () => {
  return (
    <Navbar
      brand={
        <a className="brand-logo" href="/">
          BudgetSmart
        </a>
      }
      centerLogo
      id="mobile-nav"
      style={{ backgroundColor: '#ba68c8' }}
    ></Navbar>
  );
};

export default Header;
