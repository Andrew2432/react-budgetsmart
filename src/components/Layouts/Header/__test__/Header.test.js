import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Header from '../Header';
import { cleanup } from '@testing-library/react';

afterEach(cleanup);

test('Matches snapshot', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
