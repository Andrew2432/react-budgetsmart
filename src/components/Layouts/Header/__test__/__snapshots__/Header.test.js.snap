// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches snapshot 1`] = `
Array [
  <nav
    className=""
    style={
      Object {
        "backgroundColor": "#ba68c8",
      }
    }
  >
    <div
      className="nav-wrapper"
    >
      <a
        className="brand-logo brand-logo center"
        href="/"
      >
        BudgetSmart
      </a>
      <a
        className="sidenav-trigger"
        data-target="mobile-nav"
        href="#!"
      >
        <i
          className="material-icons"
        >
          menu
        </i>
      </a>
      <ul
        className="hide-on-med-and-down"
      />
    </div>
  </nav>,
  <ul
    className="sidenav"
    id="mobile-nav"
  />,
]
`;
